<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyDialogClass</class>
 <widget class="QDialog" name="MyDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>125</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>125</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyDialog</string>
  </property>
  <widget class="QLabel" name="labelIcon">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>50</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelContent">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
